packages/toolpad-app/src/appDom/migrations/v4.ts

```diff
@@ -0,0 +1,40 @@
+import invariant from 'invariant';
+import * as appDom from '..';
+import { mapValues } from '../../utils/collections';
+import config from '../../config';
+
+export const DEFAULT_TEMPLATE_DATA_URL = new URL('/static/employees.json', config.externalUrl).href;
+
+function replaceDefaultDataStaticPath(node: appDom.AppDomNode): appDom.AppDomNode {
+  if (node.type === 'query' && node.name === 'getUsers' && node.attributes.query) {
+    return {
+      ...node,
+      attributes: {
+        ...node.attributes,
+        query: {
+          ...node.attributes.query,
+          value: {
+            ...node.attributes.query.value,
+            url: {
+              type: 'const',
+              value: DEFAULT_TEMPLATE_DATA_URL,
```

Comment:
We should probably only replace the URL if it's using `/static/employees.json` specifically - so that if someone creates a query called `getUser` (pretty common name also) it doesn't get replaced.

Reply:
Yep, good catch there

Reply:
I feel like this is not what migrations are for. Can't we just replace it in the template?

Reply:
e.g. when we load the default template
```tsx
const defaultTemplate = await readJsonFile(path.resolve(__dirname, './default.json'))
defaultTemplate.nodes['1313wn3'].attributes.query.value.url.value = DEFAULT_TEMPLATE_DATA_URL
return defaultTemplate
```
and be done with it